src/main/java/io/github/moulberry/notenoughupdates/miscgui/itemcustomization/GuiItemCustomize.java

```diff
@@ -0,0 +1,1050 @@
+			int leatherColour = ItemCustomizationUtils.getLeatherColour(this);
+
+			String text = "§b§lCustom Leather Colour";
+			boolean reset = true;
+
+			if (!animatedLeatherColours.isEmpty()) {
+				text = "§b§lOverridden by Animated";
+				reset = false;
+			}
+
+			ItemCustomizationUtils.renderColourBlob(xCenter, yTop, leatherColour, text, reset);
+
+			yTop += 20;
+		}
+
+		if (!lastCustomItem.equals(textFieldCustomItem.getText())) {
+			updateData();
+		}
+		lastCustomItem = textFieldCustomItem.getText();
+
+		int offset = 200;
+		if (!supportCustomLeatherColour) offset -= 20;
+		if (!enchantGlint) offset -= 16;
+
+		ItemCustomizationUtils.renderTextBox(textFieldCustomItem, "§7Enter Custom Item ID...", xCenter - textFieldCustomItem.getWidth() / 2 - 10 + 11, yTopText + offset, 180);
+
+		if (supportCustomLeatherColour) {
+			yTop += 25;
+			ItemCustomizationUtils.renderFooter(xCenter, yTop, guiType);
+		}
+
+		renderHeight = yTop - yTopStart;
+
+		if (editor != null) {
+			editor.render();
+		}
+
+		if (tooltipToDisplay != null) {
+			Utils.drawHoveringText(tooltipToDisplay, mouseX, mouseY, width, height, -1);
+		}
+
+		super.drawScreen(mouseX, mouseY, partialTicks);
+	}
+
+	private void drawScreenAnimatedDyes(int mouseX, int mouseY, float partialTicks) {
+		drawDefaultBackground();
+
+		List<String> tooltipToDisplay = null;
+
+		ScaledResolution scaledResolution = new ScaledResolution(Minecraft.getMinecraft());
+
+		int xCenter = scaledResolution.getScaledWidth() / 2;
+		int yTopStart = (scaledResolution.getScaledHeight() - renderHeight) / 2;
+		int yTop = yTopStart;
+
+		renderHeader(xCenter, yTop);
+
+		ItemCustomizationUtils.renderPresetButtons(xCenter,
+			yTop,
+			ItemCustomizationUtils.validShareContents("NEUCUSTOMIZE"),
+			true,
+			"preset"
+		);
+		ItemCustomizationUtils.renderPresetButtons(xCenter,
+			yTop + 50,
+			ItemCustomizationUtils.validShareContents("NEUANIMATED"),
+			!this.animatedLeatherColours.isEmpty(),
+			"animated"
+		);
+
+		yTop += 14;
+
+		renderBigStack(xCenter, yTop);
+
+		yTop += 115;
+
+		int adjustedY = yTop + pageScroll + 20;
+
+		for (int i = 0; i < animatedLeatherColours.size(); i++) {
+			if (adjustedY + 20 * i < yTopStart + 130 || adjustedY + 20 * i >= yTopStart + guiScaleOffset() + 150) {
+				continue;
+			}
+
+			int leatherColour = ItemCustomizationUtils.getLeatherColour(animatedLeatherColours.get(i));
+			ItemCustomizationUtils.renderColourBlob(xCenter, yTop, leatherColour, "§b§lDye Colour " + (i + 1), true);
+
+			yTop += 20;
+		}
+
+		Minecraft.getMinecraft().getTextureManager().bindTexture(PLUS);
+		GlStateManager.color(1, 1, 1, 1);
+		RenderUtils.drawTexturedRect(xCenter + 90 - 12, yTop + 4, 10, 10, GL11.GL_NEAREST);
+
+		ItemCustomizationUtils.renderTextBox(textFieldTickSpeed,
+			"§7Speed...",
+			xCenter - textFieldCustomItem.getWidth() / 2 - 10 + 11,
+			yTop,
+			45
+		);
+
+		Utils.renderShadowedString("§c§lClear", xCenter - 20, yTop + 6, xCenter * 2);
```

Comment:
I think this should have a button background, similar to the other buttons that are there (Main Menu, Hypixel Dyes)